rdrf/rdrf/settings.py

```diff
@@ -239,7 +239,12 @@
 
 # valid values django.core.files.storage.FileSystemStorage and
 # storages.backends.database.DatabaseStorage
-DEFAULT_FILE_STORAGE = env.get("storage_backend", "django.core.files.storage.FileSystemStorage")
+# DEFAULT_FILE_STORAGE = env.get("storage_backend", "django.core.files.storage.FileSystemStorage")
+if env.get("storage_backend", "fs") == "db":
```

Comment:
A bit misleading

If someone were to set the environment field storage_backend to , say , "s3", it would use FileSystemStorage

Reply:
Better to make the actual behaviour fit what is input or error